src/services/__tests__/TeamAssignmentService.unit.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { createRandomTeams } from "../TeamAssignmentService";
+import { Team } from "../../types/common";
+import { BallChaserQueueBuilder } from "../../../.jest/Builder";
+
+const mockBallChasers = BallChaserQueueBuilder.many(6);
+
+//Remove team values and mmr as they will be set below.
+mockBallChasers.forEach((player) => (player.team = null));
```

Comment:
Use the override built into the `many` builder instead.

Reply:
Fixed